lib/modules/manager/gradle/extract/catalog.ts

```diff
@@ -207,7 +211,7 @@ function extractDependency({
 export function parseCatalog(
   packageFile: string,
   content: string
-): PackageDependency<GradleManagerData>[] {
+): [PackageDependency<GradleManagerData>[], string[]] {
```

Comment:
I would like to return `{ deps, urls, vars }` structure, just like `parseProps` and `parseGradle` do